src/util.ts

```diff
@@ -19,6 +19,12 @@ const DERIVATION_ARTEFACTS_CURRENT_VERSION = 'K';
 export const encode64 = util.encode64;
 export const decode64 = util.decode64;
 export const encodeUtf8 = util.encodeUtf8;
+export const utf8ToBytes = util.text.utf8.encode;
+export const bytesToUtf8 = util.text.utf8.decode;
+export const utf16ToBytes = util.text.utf16.encode;
+export const bytesToUtf16 = util.text.utf16.decode;
+export const binaryStringToBytes = util.binary.raw.decode;
+export const bytesToBinaryString = util.binary.raw.encode;
 
 export const generateRandomKey = (length = 32) => random.getBytesSync(length);
```

Comment:
I think this method might as well return an `EncryptionKey` instance now. If we still want to support random bytes we can add that as an appropriately named method.

```suggestion
export const generateRandomKey = (length = 32) => EncryptionKey.fromBytes( stringAsBinaryBuffer(generateRandomBytes) );
export const generateRandomBytes = (length = 32) => random.getBytesSync(length);
```